### DISCUSSION TITLE
Scatter plot density

### DISCUSSION THREAD

**Discussion by daniel-caichac-DHI** [DHI/modelskill#119] [Ideas]
Hi all,

I've been trying to change/add an option of the scatter plots from gray dots with a 2D histogram on the back (see next fig)  
![image](https://user-images.githubusercontent.com/97288080/153578745-dc5f4ad2-c428-4774-9969-47dc8cf1a806.png)

to points painted by density (see following work in progress)
![image](https://user-images.githubusercontent.com/97288080/153580076-f2f8eb01-5a68-4f9c-ba01-68ea5f59788a.png)

however I would require to add a package to fmskill, `from scipy import interpolate` 

would that be the acceptable (@ecomodeller)  ?

**Comment by ecomodeller** [comment]
Yes you can add scipy. :shipit: 

Since mikeio is a dependency of fmskill and scipy is a dependency of mikeio, thus scipy is a transitive dependency of fmskill. 🤓 
https://github.com/DHI/mikeio/blob/82760a899f4294bdddab6a0eb18f34b0754b21e0/setup.py#L13
